lib/src/main/resources/bag/metadata/prov/provenance.xsd

```diff
@@ -115,8 +115,8 @@
     <xs:complexType name="ContentType">
         <xs:complexContent>
             <xs:extension base="dcterms:elementOrRefinementContainer">
-                <xs:choice>
-                    <xs:element ref="amd:depositorId" minOccurs="0"/>
+                <xs:choice maxOccurs="unbounded">
+                    <xs:element name="depositorId" form="unqualified" type="xs:NCName" minOccurs="0"/>
```

Comment:
why this change? We should keep this xsd in sync with the published one in easy-schema, as that will be used by the dd-validate-dans-bag